CHANGELOG.md

```diff
@@ -6,8 +6,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+- Armv8.1-M PACBTI library variants (#489).
+
+### Changed
+- -nostartfiles required when using crt0-semihosting (#460).
+
+## [19.0.0]
+
+### Added
+- AArch32 v8-A and v8-R can now be targeted (#484) (#486).
+- Experimental support for building llvm-libc overlay package (#452).
+- Additional library variants for Armv7 (#464) (#491) (#494).
+- Additional multilib mappings for v7ve and Thumb target triples (#479) (#481).
+- AArch64 v8-R library variant (#468).
```

Comment:
That patch just changed the name of the v8-A variants in preparation for adding v8-R, the latter hasn't been done yet.

Reply:
Thanks, I've moved that to changed.